health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/utils/appConfigHelpers.js

```
@@ -14,7 +14,7 @@ const getTypeAndMetaData = (field, fieldTypeMasterData = []) => {
   }
 
   // Start with the fieldType as type
-  let result = { type: matched.fieldType, appType: matched.type };
+  let result = { type: matched.fieldType, appType: matched.type, };
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'result' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)